Gopkg.toml

```diff
@@ -130,7 +130,7 @@ required = [
 
 [[override]]
   name = "github.com/ugorji/go"
-  revision = "ded73eae5db7e7a0ef6f55aace87a2873c5d2b74"
+  revision = "v1.1.1"
```

Comment:
This should most likely be `version = "1.1.1"`.

Reply:
You can't really compare output of a `dep status` with syntax used in `Gopkg.toml` file.

My initial comment was more about coming from official documentation for `Gopkg.toml` and it's `version` property: https://golang.github.io/dep/docs/Gopkg.toml.html#version

And if it allows having both cases. Why go for inconsistency if there is no other `version` property with `v` prefix in current `Gopkg.toml`. Inconsistency creates confusion. Either put this without prefix or put all others with prefix.

Reply:
That's a good explanation and I'll fix this one. Thanks @ondrej-fabry!

Reply:
OK, something weird happened. When I changes this to `1.1.1` instead of `v1.1.1`, a run of `dep ensure -update` failed because it could no longer match the version of `github.com/ugorji/go` it needed for etcd-3.3.9. I wonder if this is related to their usage of glide.